web/src/components/GeneralHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+
+import HeaderImage from './HeaderImage'
+import Contact from './Contact'
+
+import { OptionalContainer } from 'components/Container'
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    background: theme.palette.primary.main,
+    borderBottomWidth: 2,
+    borderBottomStyle: 'solid',
+    borderBottomColor: theme.palette.secondary.main
+  },
+  container: () => ({
+    display: 'flex',
+    alignItems: 'center',
+    maxWidth: '100%'
+  }),
+  title: {
+    color: theme.palette.primary.contrastText,
+    fontSize: '1.7rem'
+  },
+  titleWrapper: {
+    display: 'flex',
+    alignItems: 'center'
+  }
+}))
+
+interface Props {
+  title: string
+  productName: string
+  padding?: string
+  spacing: number
+}
+
+export const GeneralHeader: React.FunctionComponent<Props> = (props: Props) => {
+  const { title, productName, spacing, padding } = props
+  const classes = useStyles(props)
+
+  return (
+    <nav className={classes.root}>
+      <OptionalContainer style={{ paddingLeft: padding }} className={classes.container}>
```

Comment:
Can we add this `paddingLeft` to the `container` class? It would make CSS harder to reason about with both inline styles and classes.

Reply:
Is there a way to do this while keeping it passed in through props?

Reply:
Yep, check out: https://github.com/bcgov/wps/blob/e911111ec6000d49297d85ab8334762d7c2480a8/web/src/features/fireWeather/components/ExpandableContainer.tsx#L18